bin/build-locales

```diff
@@ -46,7 +46,7 @@ poFiles.forEach((pofile) => {
   // Add the moment locale JS into our locale file, if one is available and
   // we're building for AMO (which is the only app that uses moment right
   // now).
-  if (appName === 'amo') {
+  if (false && appName === 'amo') {
```

Comment:
This was added to avoid the moment code being unserialisable as JSON. I believe we've changed how we setup moment but it might be a `require` still so some extra work may still be required.